_posts/en/newsletters/2023-04-05-newsletter.md

```diff
@@ -0,0 +1,143 @@
+    [gist][segura watchtowers gist], Segura outlines several different
+    accumulator constructions worth considering.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.16.0-beta.rc5][] is a release candidate for a new major
+  version of this popular LN implementation.
+
+- [BDK 1.0.0-alpha.0][] is a test release of the major changes to BDK
+  described in [Newsletter #243][news243 bdk].  Developers of
+  downstream projects are encouraged to begin integration testing.
+
+<!-- FIXME:harding to update Tuesday -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Core Lightning #5967][] add listclosedchannels FIXME:bitschmidty
+
+- [Eclair #2566][] adds support for accepting offers.  Offers must be
+  registered by a plugin that provides a handler for responding to
+  invoice requests related to the offer and either accepting or
+  rejecting payments to that invoice.  Eclair ensures requests and
+  payments satisfy the protocol requirements---the handler only needs to
+  decide whether the item or service being purchased can be provided.
+  This allows code for marshaling offers to become arbitrarily complex
+  without affecting Eclair's internal logic.
+
+- [LDK #2062][] implements [BOLTs #1031][] (see [Newsletter
+  #226][news226 bolts1031]), [#1032][bolts #1032] (see [Newsletter
+  #225][news225 bolts1032]), and [#1040][bolts #1040], allowing the
+  ultimate receiver of a payment ([HTLC][topic htlc]) to accept a
+  greater amount than they requested and with a longer time before it
+  expires than they requested.  This makes it more difficult for a
+  forwarding node to determine that the next hop is the receiver by
+  slightly tweaking a payment’s parameters.  The merged PR also allows a
+  spender to pay a receiver slightly more than the requested amount when
+  using [simplified multipath payments][topic multipath payments]. This
+  provides the above benefit and may also be required in the case where
+  the chosen payment paths use channels with a minimum routable amount.
+  For example, Alice wants to split a 900 sat total into two parts, but
+  both of the paths she chooses require 500 sat minimum amounts. With
+  this specification change, she can now send two 500 sat payments,
+  choosing to overpay by a total of 100 sats in order to use her
```

Comment:
```suggestion
  choosing to overpay by a total of 100 sats to use her
```

Reply:
Leaving this as-is; "in order to" just feels better to me here.  :shrug: